tdesign/src/themes/muiTheme.ts

```diff
@@ -188,44 +190,53 @@ export const muiTheme = ({
         "-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;",
       // Default to sans-serif, b/c of all the text styles 2 are serif and the rest sans. List from systemfontstack.com -> sans-serif
       title1: {
```

Comment:
btw. I believe it is worth adding a general comment above all those typography-related styles, that these correspond to typography styles that are used on the design mockups. It's something that may not be obvious just by reading the code, and something that we found out during the course of this PR thanks to @onpaw